DevSecOps-WoW.md

```diff
@@ -0,0 +1,25 @@
+# CDC ReportStream DevSecOps Ways of Working
```

Comment:
I think this document would be more appropriate in the team sharepoint? If you agree, I can move it there once everyone has approved the PR.

Reply:
Sure, we can move it once we have a quorum.  I don't have access to SP yet, so I pushed here to open the WoW to discussion to erasure everyone is comfortable with what I'm proposing on behalf of my team.

Reply:
The WoW has relative hrefs to the issue template, it'll need to be updated so the links work from SP.

Reply:
Hi @MichaelEsuruoso 
1. Our WiP limit will be what we set it to.  Before we can set it, we need to run a few sprints to get realistic metrics.  We can always stay within the WiP limits as we can deprioritize work that's being replaced with higher priority item.   

2.  That's where the project engineering lead steps in and helps establish priorities.  We can work on multiple items concurrently.